sdk/Cargo.toml

```
@@ -587,20 +587,20 @@ path = "examples/wallet/output_consolidation.rs"
 required-features = [ "wallet", "stronghold" ]
 
 [[example]]
-name = "ping"
-path = "examples/wallet/ping.rs"
-required-features = [ "wallet", "storage" ]
-
-[[example]]
-name = "pong"
-path = "examples/wallet/pong.rs"
+name = "multithread"
+path = "examples/wallet/multithread.rs"
 required-features = [ "wallet", "storage" ]
 
 [[example]]
 name = "recover_accounts"
 path = "examples/wallet/recover_accounts.rs"
 required-features = [ "wallet" ]
 
+[[example]]
+name = "spammer"
+path = "examples/wallet/spammer.rs"
+required-features = [ "wallet", "storage" ]
```

```suggestion
required-features = [ "wallet" ]
```
Can we remove this and `.with_storage_path(&var("WALLET_DB_PATH").unwrap())` in the example?

---

Actually I use this in every example to create a wallet db with an `example-` prefix.